thriftpy/thrift.py

```diff
@@ -110,10 +112,11 @@ def __ne__(self, other):
 
 
 class TClient(object):
+
     def __init__(self, service, iprot, oprot=None):
         self._service = service
         self._iprot = self._oprot = iprot
-        if oprot is not None:
```

Comment:
is the change ok here? @maralla  some parameters  I dont really know what its types will be and In line 173 too